static/app/components/workflowEngine/form/control/priorityControl.tsx

```diff
@@ -13,43 +13,45 @@ import {IconArrow, IconChevron} from 'sentry/icons';
 import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 import {PriorityLevel} from 'sentry/types/group';
+import {
+  METRIC_DETECTOR_FORM_FIELDS,
+  useMetricDetectorFormField,
+} from 'sentry/views/detectors/components/forms/metricFormData';
 
 function ThresholdPriority() {
-  const lowThresholdDirection = useFormField<string>('conditionGroup.conditions.0.type')!;
-  const lowThreshold = useFormField<string>('conditionGroup.conditions.0.comparison')!;
+  const conditionType = useMetricDetectorFormField(
+    METRIC_DETECTOR_FORM_FIELDS.conditionType
+  );
+  const conditionValue = useMetricDetectorFormField(
+    METRIC_DETECTOR_FORM_FIELDS.conditionValue
+  );
   return (
     <div>
-      {lowThresholdDirection === ''
-        ? t('Above')
-        : lowThresholdDirection === 'above'
-          ? t('Above')
-          : t('Below')}{' '}
-      {lowThreshold === '' ? '0s' : lowThreshold + 's'}
+      {conditionType === 'gt' ? t('Above') : t('Below')}{' '}
+      {conditionValue === '' ? '0s' : conditionValue + 's'}
     </div>
   );
 }
 
 function ChangePriority() {
-  const lowThresholdDirection = useFormField<string>('conditionGroup.conditions.0.type')!;
+  const conditionType = useMetricDetectorFormField(
+    METRIC_DETECTOR_FORM_FIELDS.conditionType
+  );
   const lowThreshold = useFormField<string>('conditionGroup.conditions.0.comparison')!;
   return (
     <div>
       {lowThreshold === '' ? '0' : lowThreshold}%{' '}
-      {lowThresholdDirection === ''
-        ? t('higher')
-        : lowThresholdDirection === 'higher'
-          ? t('higher')
-          : t('lower')}
+      {conditionType === 'gt' ? t('higher') : t('lower')}
```

Comment:
should we cache consistent translations like `higher` and `lower` in a constants file?

Reply:
usually ends up not being worth it because of casing and stuff or you want 'higher than' in another file